dynamic/image/src/main/java/com/twocoders/dynamic/image/BaseDynamicImage.kt

```diff
@@ -0,0 +1,118 @@
+package com.twocoders.dynamic.image
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.os.Parcel
+import android.os.Parcelable
+import android.widget.ImageView
+import androidx.annotation.CallSuper
+import androidx.annotation.ColorInt
+import androidx.annotation.DrawableRes
+import com.twocoders.dynamic.image.component.UriComponent
+import com.twocoders.extensions.common.NO_ID
+import com.twocoders.extensions.common.getDrawable
+import com.twocoders.extensions.common.logd
+
+/**
+ *
+ * Handy class which can be used to bind image data to views using them.
```

Comment:
```suggestion
 * Handy class which can be used to bind image data to views.
```

to using them tam moc nedava vyznam. koho them?